src/absil/il.fs

```diff
@@ -4504,17 +4504,17 @@ type ILGlobals with
         | Some res -> res
 
     member this.mkDebuggerStepThroughAttribute() = mkILCustomAttribute this (mkSystemDiagnosticsDebugTypeRef this tname_DebuggerStepThroughAttribute, [], [], [])
-    member this.mkDebuggableAttribute (jitTracking, jitOptimizerDisabled) =
-        mkILCustomAttribute this (mkSystemDiagnosticsDebuggableTypeRef this, [this.typ_Bool; this.typ_Bool], [ILAttribElem.Bool jitTracking; ILAttribElem.Bool jitOptimizerDisabled], [])
+    member this.mkDebuggableAttribute (jitOptimizerDisabled) =
+        mkILCustomAttribute this (mkSystemDiagnosticsDebuggableTypeRef this, [this.typ_Bool; this.typ_Bool], [ILAttribElem.Bool false; ILAttribElem.Bool jitOptimizerDisabled], [])
 
 
-    member this.mkDebuggableAttributeV2(jitTracking, ignoreSymbolStoreSequencePoints, jitOptimizerDisabled,enableEnC) =
+    member this.mkDebuggableAttributeV2(ignoreSymbolStoreSequencePoints, jitOptimizerDisabled, enableEnC) =
         let tref = mkSystemDiagnosticsDebuggableTypeRef this
         mkILCustomAttribute this 
           (tref,[mkILNonGenericValueTy (tref_DebuggableAttribute_DebuggingModes this)],
-           [ILAttribElem.Int32( 
+           [ILAttribElem.Int32(
                             (* See System.Diagnostics.DebuggableAttribute.DebuggingModes *)
-                              (if jitTracking then 1 else 0) |||  
+                              (0) |||  
```

Comment:
Just remove this line :)  However I have to ask: would it reduce risk to leave this in, just in case some tool (e.g. Mono?) pays attention to this?